tests/melt_transport_compressible_iterative/statistics

```diff
@@ -23,8 +23,8 @@
 # 23: Minimum velocity magnitude on boundary with indicator 2 ("bottom") (m/s)
 # 24: Maximum velocity magnitude on boundary with indicator 3 ("top") (m/s)
 # 25: Minimum velocity magnitude on boundary with indicator 3 ("top") (m/s)
-0 0.000000000000e+00 0.000000000000e+00 256 2467 1089 1089 0 0 1012 5141 21562 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 
+0 0.000000000000e+00 0.000000000000e+00   0    0    0    0 0 0 1012 5141 21562 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 
 0 0.000000000000e+00 0.000000000000e+00   0    0    0    0 0 0  176  906  3756 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 
 0 0.000000000000e+00 0.000000000000e+00   0    0    0    0 0 0   29  151   626 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 
 0 0.000000000000e+00 0.000000000000e+00   0    0    0    0 0 0   29  142   629 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 
-0 0.000000000000e+00 0.000000000000e+00   0    0    0    0 0 0   27  140   585 2.23414594e-01 3.37402230e-01 5.09193136e-01 1.57518986e+00 2.96291766e+00 3.37400763e-01 1.25883388e-01 3.37400763e-01 1.25883388e-01 1.25000000e-01 1.25000000e-01 3.39785229e-01 3.39785229e-01 
+0 0.000000000000e+00 0.000000000000e+00 256 2467 1089 1089 0 0   27  140   585 2.23414594e-01 3.37402230e-01 5.09193136e-01 1.57518986e+00 2.96291766e+00 3.37400763e-01 1.25883388e-01 3.37400763e-01 1.25883388e-01 1.25000000e-01 1.25000000e-01 3.39785229e-01 3.39785229e-01 
```

Comment:
all of these lines correspond to the same time step, so they should have the same number of cells and dofs, but the first few are zero